lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
On the other hand, moving things around is not such a big deal because the user-facing API has all of them under `lasagne.layers` anyway (except for those in  `cuda_convnet`, `dnn`, `corrmm` because they have special dependencies).